### PR TITLE
pull request branch for translator

### PR DESCRIPTION
**Pull request by alekseyl**
clear PR for key translator

### CODE REVIEW

**Comment by sacry-dyn** [comment]
@alekseyl it is totally fine to not test external APIs. but test the new functionality (and mock/stub) external calls. It is more a syntax/what does the function express thing.

File: lib/slimkeyfy/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module SlimKeyfy
-  VERSION = "0.0.4"
+  VERSION = "0.0.9"
```
Comment (sacry-dyn) [inline lib/slimkeyfy/version.rb]: Looks okay. Please change to 0.1.0 (as this is a new feature)

**Comment by alekseyl** [comment]
Done!

**Comment by sacry-dyn** [comment]
@alekseyl great! Thank you very much

**Pull request merged**